internal/pkg/image/driver/imagedriver.go

```diff
@@ -346,7 +345,8 @@ func (d *fuseappsDriver) Start(params *image.DriverParams, containerPid int) err
 
 // Stop the process associated with the mount target, if there is one.
 // If kill is not true, an unmount should already have happened so at
-//   first just wait for the process to exit.
```

Comment:
I didn't spotted this at first, can you also remove the space from the comment to avoid having go 1.19 formatting it to:

```
//
//	first just wait for the process to exit.
```

Thanks !

Reply:
@Juneezee ^^ , remain just this one and it's good to go